apps/dashboard/src/routes/root.tsx

```diff
@@ -1,26 +1,32 @@
+import { ClerkProvider } from '@/components/providers/clerk-provider';
+import { SegmentProvider } from '@/components/providers/segment-provider';
 import { Outlet } from 'react-router-dom';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+
+const queryClient = new QueryClient();
 
 export default function Root() {
   return (
-    <>
-      <div className="relative min-h-dvh flex flex-col">
-        <div className="sticky left-0 top-0 h-12 w-full bg-green-200">Header</div>
-        <ul>
-          <li>
-            <a
-              href="/legacy/"
-              target="_self"
-              className="text-blue-600 visited:text-purple-600 hover:border-current hover:border-b"
-            >
-              Go to the Web App
-            </a>
-          </li>
-        </ul>
+    <QueryClientProvider client={queryClient}>
+      <ClerkProvider>
+        <SegmentProvider>
```

Comment:
✏ **suggestion:** can we maybe move this to the `Providers` component?

Reply:
I am not a big fan of this split, as we did it in the `web` project. It's better to see all of the context in the root file. If it gets too big, we will revise.